loleaflet/admin/adminLog.html

```diff
@@ -9,7 +9,18 @@ <h3 class="title is-3"><script>document.write(l10nstrings.strLog)</script></h3>
       <button type="button" id="button-open-log-list-modal" class="button is-info" style="width:120px;">
         <script>document.write(l10nstrings.strSetLogLevels)</script>
       </button>
-    </div>
+	</div>
```

Comment:
It seems this file isn't using tabs either. Please review your editor settings, it seems to prefer tabs, which isn't really very friendly (because it depends on the settings of each user's editor). Prefer spaces, unless there is a good reason not to.